packages/backend/src/config/parseConfig.ts

```diff
@@ -785,6 +786,9 @@ export const parseConfig = (): LightdashConfig => {
         },
         github: {
             appName: process.env.GITHUB_APP_NAME || 'lightdash-app-dev',
+            redirectDomain:
```

Comment:
if not GIHTUB_REDIRECT_DOMAIN specified, it will take the subdomain from the siteUrl (eg: SITE_URL=https://analytics.lightdash.cloud) 

 https://analytics.lightdash.cloud -> analytics